siliconcompiler/schema/schema_cfg.py

```diff
@@ -112,9 +112,11 @@ def scparam(cfg,
             cfg['node']['default']['default']['date'] = []
             cfg['node']['default']['default']['author'] = []
             cfg['node']['default']['default']['filehash'] = []
+            cfg['node']['default']['default']['dependency'] = []
```

Comment:
package, also need to add this to the schema documentation

Reply:
Sure but wouldn't this be confusing to the user?
The schema will say this file is part of package xyz but then the `package` information will be in the `dependency` section of the schema...

Reply:
we will likely need to make some more changes to the the schema to be self consistent, but it should be `package`